gptcache/adapter/langchain_llms.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, List
+from .adapter import adapt
+
+
+class LangChinLLMs:
```

Comment:
type error, `LangChain`

Reply:
did you mean rename `LangChinLLMs ` to `LangChain`